pkg/processor/runtime/rpc/abstract.go

```diff
@@ -231,9 +254,12 @@ func (r *AbstractRuntime) startWrapper() error {
 		r.Statistics,
 		r.configuration.WorkerID,
 		r.configuration.Mode,
+		timeout,
 	)
 
-	var err error
+	if err != nil {
+		return errors.Wrap(err, "Failed to create connection manager configuration")
+	}
```

Comment:
You don't return an error from `NewManagerConfigration`, so this `err` is still the one from `GetEventTimeout`